docs/configuration.asciidoc

```diff
@@ -692,6 +692,18 @@ The timeout is applied once the agent has sent the entire request body to the AP
 If the response from the server takes longer than allowed by this timeout,
 the HTTP request is terminated and the TCP socket closed.
 
+[[sanitize-field-names]]
+==== `sanitizeFieldNames`
+* *Type:* Array
+* *Default:* ['password', 'passwd', 'pwd', 'secret', '\*key', '*token*', '\*session*', '\*credit*', '\*card*', 'authorization', 'set-cookie']
+* *Env:* `ELASTIC_SANITIZE_FIELD_NAMES`
+
+Sometimes it is necessary to sanitize the data sent to Elastic APM, e.g. remove sensitive data.
+
+The `sanitizeFieldNames` configuration values allows you to configure a list of wildcard patterns of field names which should be _removed_ from agent payloads. These patterns apply to the request and response HTTP headers, as well as any form field capturing during an `application/x-www-form-urlencoded data` request.
```

Comment:
Does the spec necessarily say these headers should be *removed*? Or just that the sensitive data in them should be cleared. E.g. As a user I might want to know that my form data included a "passwd", even if the value of that field is "REDACTED" or whatever.

XXX

Reply:
Rejiggered things so that `sanitizeFieldNames` redacts the headers instead of removing them.